### DISCUSSION TITLE
Opposite of to_dense_batch()

### DISCUSSION THREAD

**Discussion by ldv1** [pyg-team/pytorch_geometric#4900] [Q&A] [answered]
Hi,

Is the opposite of `to_dense_batch()` implemented ? I couldn't find it, but maybe there is a version of it somewhere.
I want to do something like

```
batch_x, mask = to_dense_batch(x, batch)
# some ops
x, batch = opposite_of_to_dense_batch(batch_x, mask)
```
thanks !

**Comment by rusty1s** [comment, marked as answer]
The opposite is:
```python
x = batch_x[mask]
```
:)

**Reply by ldv1** [reply]
indeed ! thanks for your prompt response
